salem/gis.py

```diff
@@ -1455,12 +1455,13 @@ def googlestatic_mercator_grid(center_ll=None, nx=640, ny=640, zoom=12):
 
     # Meter per pixel
     mpix = (2 * np.pi * google_earth_radius) / google_pix / (2**zoom)
-    xx = nx * mpix
-    yy = ny * mpix
+    xx = nx * scale * mpix
```

Comment:
It would be cleaner to redefine nx and ny earlier with:

```python
nx = nx * scale
ny = ny * scale
```
Then you won't have to change the rest of the code I think